Cargo.toml

```diff
@@ -99,16 +99,17 @@ workspace = true
 vello = { workspace = true }
 thiserror = "1.0.69"
 usvg = "0.44.0"
-image = { version = "0.25.5", default-features = false, features = [
-  "webp",
-  "png",
-  "jpeg",
-  "gif",
-] }
+image = { version = "0.25.5", default-features = false, features = [ "webp", "png", "jpeg", "gif"], optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dev-dependencies]
 wasm-bindgen-test = "0.3.49"
 
 [features]
+default = ["image", "png", "gif", "jpeg", "webp"]
 # Enables the wgpu feature on vello, which is disabled by default
 wgpu = ["vello/wgpu"]
+image = ["dep:image"]
+png = ["image", "image/png"]
```

Comment:
Only request, I think, would be to rename these to `image_*` to help make it more clear. Or even `image_format_*` ...

Reply:
@waywardmonkeys done!